Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs

```diff
@@ -9,6 +9,10 @@ namespace Xamarin.Forms.Platform.UWP
 	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
 	{
 		bool _fontApplied;
+		FormsRadioButton button;
+		PointerEventHandler _pointerPressedHandler;
+
+		public bool IsDisposed { get; private set; }
```

Comment:
No need for this to be public; a private `_disposed` member would be sufficient.